gsoc/models.py

```diff
@@ -0,0 +1,205 @@
+from django.db import models
+from eventtools.models import BaseEvent, BaseOccurrence
+
+
+class Gsoc(models.Model):
+    count = models.IntegerField(primary_key=True)
+
+    def __str__(self):
+        return str(self.count)
+
+
+class Program(models.Model):
+    year = models.IntegerField(primary_key=True)
+    state = models.CharField(max_length=100)
+    is_current = models.BooleanField()
+    org_signup_open = models.BooleanField()
+    org_approval = models.BooleanField()
+    orgs_published = models.BooleanField()
+    student_signup_open = models.BooleanField()
+    slot_request = models.BooleanField()
+    slot_allocation_grace_period = models.BooleanField()
+    project_acceptance = models.BooleanField()
+    project_decisions_finalization = models.BooleanField()
+    first_work_period = models.BooleanField()
+    first_evaluations = models.BooleanField()
+    second_work_period = models.BooleanField()
+    second_evaluations = models.BooleanField()
+    third_work_period = models.BooleanField()
+    final_week = models.BooleanField()
+    final_evaluations_mentor = models.BooleanField()
+    post_program = models.BooleanField()
+    org_signup_open = models.OneToOneField('GsocEvent', related_name='org_signup_open')
```

Comment:
Like "Mentor", put the GsocEvent and GsocOccurence above `Program`, so you can refer to the object instead of the name.